sdk/cognitiveservices/cognitiveservices-luis-runtime/src/operations/predictionOperations.ts

```diff
@@ -1,8 +1,4 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- *
```

Comment:
Please add the copyright text back manually

Reply:
I am sorry for missing all of that